pkg/manifest/platform_linux.go

```diff
@@ -14,6 +14,9 @@
 
 package manifest
 
-func platform() string {
+// CurrentPlatform is the value of Reference.Platform for this GOOS.
+// Specifically, when "getenvoy" runs and detects GOOS == "linux", we choose the "linux-glibc" Envoy platform.
```

Comment:
this broke my brain. I'm not sure the history of this choice, but in any case it will probably chance when upstream does. For now, this comment can hopefully save the next person.